src/server/apis/issues.ts

```diff
@@ -1,99 +1,138 @@
+  }
 
-    const assignedIssuesResult = await github().query<AssignedIssuesQuery>({
-      query: assignedIssuesQuery,
-      variables: {
-        query: `assignee:${assigneeLogin} is:issue state:open`,
-      },
-      fetchPolicy: 'network-only',
-      context: {token: userRecord.githubToken}
-    });
-
-    const issues: Issue[] = [];
-    if (assignedIssuesResult.data.search.nodes) {
-      for (const node of assignedIssuesResult.data.search.nodes) {
-        if (!node) {
-          continue;
-        }
+  const issueResult = await github().query<IssuesSearchQuery>({
+    query: issueQuery,
+    variables: {
+      query: queryCb(assigneeLogin),
+    },
+    fetchPolicy: 'network-only',
+    context: {token: userRecord.githubToken}
+  });
+
+  const issues: Issue[] = [];
+  if (issueResult.data.search.nodes) {
+    for (const node of issueResult.data.search.nodes) {
+      if (!node) {
+        continue;
+      }
 
-        if (node.__typename !== 'Issue') {
-          continue;
-        }
+      if (node.__typename !== 'Issue') {
+        continue;
+      }
 
-        if (!node.author) {
-          // This should never be the case
-          continue;
-        }
+      if (!node.author) {
+        // This should never be the case
+        continue;
+      }
 
-        let hasNewActivity = false;
-        if (lastViewedInfo && loginRecord) {
-          const lastActivity = await getIssueLastActivity(assigneeLogin, node);
-          if (lastActivity) {
-            hasNewActivity = await issueHasNewActivity(
-                loginRecord, lastActivity, lastViewedInfo[node.id]);
-          }
+      let hasNewActivity = false;
+      if (lastViewedInfo) {
+        const lastActivity = await getIssueLastActivity(assigneeLogin, node);
+        if (lastActivity) {
+          hasNewActivity = await issueHasNewActivity(
+              userRecord, lastActivity, lastViewedInfo[node.id]);
         }
-
-        issues.push({
-          id: node.id,
-          title: node.title,
-          repo: node.repository.name,
-          owner: node.repository.owner.login,
-          author: node.author.login,
-          avatarUrl: node.author.avatarUrl,
-          createdAt: new Date(node.createdAt).getTime(),
-          url: node.url,
-          popularity: fetchPopularity(node),
-          hasNewActivity,
-        });
       }
+
+      issues.push({
+        id: node.id,
+        title: node.title,
+        repo: node.repository.name,
+        owner: node.repository.owner.login,
+        author: node.author.login,
+        avatarUrl: node.author.avatarUrl,
+        createdAt: new Date(node.createdAt).getTime(),
+        url: node.url,
+        popularity: fetchPopularity(node),
+        hasNewActivity,
+        status: statusCb(node, assigneeLogin),
+      });
     }
+  }
+
+  const issuesResponse: IssuesResponse = {
+    issues: issues.sort((a: Issue, b: Issue) => {
+      if (a.hasNewActivity === b.hasNewActivity) {
```

Comment:
This sorts all the assigned issues too right? Can we instead change this to be sorted by `lastActivity`? This would mean ordering is not based on the dots, but it makes it very likely that dots appear at the top. This will result in the UI not jumping around if you read an item, then refresh etc.